src/com/xilinx/rapidwright/edif/EDIFTools.java

```diff
@@ -980,12 +980,28 @@ public static EDIFNetlist createNewNetlist(String topName, boolean addINV){
 	 * @return The newly created netlist from the provided cell inst.
 	 */
 	public static EDIFNetlist createNewNetlist(EDIFCellInst cellInst){
+	    List<String> librariesOrder = new ArrayList<>(cellInst.getCellType().getNetlist().getLibrariesMap().keySet());
```

Comment:
Am I missing where `cellInst.getCellType().getNetlist()` is modified, since I can see that it is copied into the netlist `n` but not where it would be modified?

Reply:
Not sure why we want to modify the source netlist?  We are just copying from it.

Reply:
Right. You copy this netlist's library order here, and my assumption was that you did this because you were worried it would change before/if you needed it later inside the `if` below?

Reply:
Thanks, I've updated it and the erroneous Javadoc that I think polluted my thinking.